### DISCUSSION TITLE
Seam Internal Perimeter not aligned to Exteral Perimeter

### DISCUSSION THREAD

**Discussion by Yamh631** [supermerill/SuperSlicer#3835] [Q&A]
**Version : SuperSlicer_2.4.58.5_win64_220924
Project : [35mm.zip](https://github.com/supermerill/SuperSlicer/files/11420484/35mm.zip)**

Hi,

With these parameters :

![Seam custom angle cost 100 travel cost 0 bis](https://user-images.githubusercontent.com/35656304/236649557-85ef7c43-7e11-4fe4-a439-3acec128fdd7.png)

The seams External/Internal Perimeters are aligned ("seam gap" parameter was increased to see the Seams in the photo) 
If I use the Seam-sphere (Right click on Object-> Add Seam Position ), the perimeters are well aligned too.

![Seam custom angle cost 100 travel cost 0](https://user-images.githubusercontent.com/35656304/236649563-d8eebde1-ee0d-4b13-9e93-6410d3dad87a.png)

**--------------------------
AND
--------------------------**

With these parameters :

![Seam Rear bis](https://user-images.githubusercontent.com/35656304/236649678-f546356a-3222-4142-bd37-715ac5dbfeb3.png)

The seams External/Internal Perimeters **are not** aligned ("seam gap" parameter was increased to see the Seams in the photo) 

![Seam Rear](https://user-images.githubusercontent.com/35656304/236649680-eb4f7d49-4c51-4abd-8546-39dca5c9a835.png)

**Why the internal perimeter does not follow the external perimeter with this parameter ?**

Best Regards

**Comment by neophyl** [comment]
Without a project to see all of your settings its almost impossible to debug.  There are so many that all interact.  Please save your project (File>Save Project as) and attach it here.   Then maybe someone can figure out what is happening.

**Reply by Yamh631** [reply]
Thanks I updated the subject with Project [35mm.zip](https://github.com/supermerill/SuperSlicer/files/11420498/35mm.zip)
